nodes/mtdevice.py

```diff
@@ -1254,6 +1259,87 @@ def usage():
             For longitude, latitude, altitude and orientation (on MTi-G-700):
                 "pl400fe,pa400fe,oq400fe"
 
+Synchronization settings:
+    The format follows the xsens protocol documentation. All fields are required
+    and separated by commas.
+    Note: The entire synchronization buffer is wiped every time a new one 
+          is set, so it is necessary to specify the settings of multiple 
+          lines at once.
+
```

Comment:
You should also say how to clear the synchronization settings. Better yet, why not a keyword `clear` as in `SetUTCTime`? (which would send a single sync setting with polarity 0.)